ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14-periodics.yaml

```diff
@@ -23350,18 +23350,7 @@ periodics:
     ci.openshift.io/generator: prowgen
     job-release: "4.14"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-openshift-logging-6.0-gcp-ipi-sdn-fips-f7-logging
-  reporter_config:
-    slack:
-      channel: '#aos-qe-logging'
```

Comment:
Are those removal expected ?

Reply:
No

Reply:
@QiaolingTang  Do you remember when this reporter_config are added? Shall we added it manual for all Logging Jobs?

Reply:
@liangxia  Fixed by add slack notification manually. PTAL